packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -260,7 +261,11 @@ const RealAccountSignup = ({
         if (getActiveModalIndex() === modal_pages_indices.status_dialog) return 'auto';
         if (!currency) return '688px'; // Set currency modal
         if (has_real_account && currency) {
-            if (show_eu_related_content && getActiveModalIndex() === modal_pages_indices.add_or_manage_account) {
+            if (
+                show_eu_related_content &&
+                !(is_pre_appstore && is_low_risk) &&
```

Comment:
@mahdiyeh-deriv it's because if we just have `show_eu_related_content` then height for modal set as 420px because for EU users cryptocurrencies are not supported, but in our scenario the user is not from EU but have both types of accounts (for EU and Non-EU)